packages/GT/GT.0.2.0/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+
+name: "GT"
+synopsis: "Generic programming with extensible transformations"
+description: """
+Yet another library for generic programming. Provides syntax extensions
+both for camlp5 and PPX which allow decoration of type declarations with
+following compile-time code generation. Provides the way for creating
+plugins (compiled separately from the library) for enchancing supported
+type transformations.
+
+Strongly remids the `visitors` library from François Pottier.
+During desing of a library of these kind there many possible
+design decision and in many cases we decided to implement
+the decision opposite to the one used in `visitors`.
+"""
+version: "0.2.0"
+
+maintainer: "kakadu.hafanana@gmail.com"
+authors: ["https://github.com/dboulytchev" "https://github.com/Kakadu"]
+homepage: "https://github.com/JetBrains-Research/GT"
+bug-reports: "https://github.com/JetBrains-Research/GT/issues"
+depends: [
+  "ocaml"
+  "ocamlfind"  {build}
+  "ocamlbuild" {build}
+  "ocamlgraph" {build}
```

Comment:
```suggestion
  "ocamlgraph"
```
According to your META file, ocamlgraph is required as a library, it is not a build dependency.